lib/services/devices/devices-NGSI-LD.js

```diff
@@ -275,7 +275,7 @@ function updateEntityNgsiLD(deviceData, updatedDevice, callback) {
  *
  * @param {Object} deviceObj                    Object with all the device information (mandatory).
  */
-function updateRegisterDeviceNgsiLD(deviceObj, callback) {
+function updateRegisterDeviceNgsiLD(deviceObj, typeNameUpdated, callback) {
```

Comment:
I understand `typeNameUpdate` means "type and name [=id] of the entity update". Maybe a better name could be entityUpdated or entityInfoUpdated ("typeName" is less clear, from my point of view).

Reply:
Fixed in 2cd4d32